packages/dashboard/src/assets/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Manage local static resource
+ */
+export const LOCAL_IMAGE = {
```

Comment:
this design no can tree shaking.

plz unpack this object, direct expose `ABOUT_DIALOG_BACKGROUND `